poc/suite_secp256k1.sage

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/sage
+# vim: syntax=python
+
+import hashlib
+import sys
+try:
+    from sagelib.common import sgn0_le
+    from sagelib.h2c_suite import BasicH2CSuiteDef, BasicH2CSuite, IsoH2CSuiteDef, IsoH2CSuite
+    from sagelib.svdw_generic import GenericSvdW
+    from sagelib.sswu_generic import GenericSSWU
+    from sagelib.suite_p256 import _test_suite
+    from sagelib.iso_values import iso_secp256k1
+except ImportError:
+    sys.exit("Error loading preprocessed sage files. Try running `make clean pyfiles`")
+
+p = 2^256 - 2^32 - 2^9 - 2^8 - 2^7 - 2^6 - 2^4 - 1
+F = GF(p)
+A = F(0)
+B = F(7)
+Ap = F(0x3f8731abdd661adca08a5558f0f5d272e953d363cb6f0e5d405447c01a444533)
```

Comment:
add a comment to recall the reader that Ap, Bp are parameters of the curve isogenous to E.